frontend/src/features/mapSideBar/SidebarContents/subforms/ClassificationForm.tsx

```diff
@@ -86,17 +90,37 @@ export const ClassificationForm: React.FC<IClassificationFormProps> = ({
   encumbranceField,
 }) => {
   const formikProps = useFormikContext();
+  const keycloak = useKeycloakWrapper();
   let surplusActiveOrEncumbered =
     getIn(formikProps.values, field) === Classifications.SurplusEncumbered ||
     getIn(formikProps.values, field) === Classifications.SurplusActive;
 
   /** classId based on current formik values to determine which classsification information box to display */
   let classId = getIn(formikProps.values, field);
 
+  /** determine how to filter the classifications */
+  const determineFilter = (val: Number) => {
+    /** non SRES users cannot see subdivided */
+    if (!keycloak.hasClaim(Claims.ADMIN_PROPERTIES)) {
+      return (
+        val !== Classifications.Subdivided &&
+        val !== Classifications.Disposed &&
+        val !== Classifications.Demolished
+      );
+    } else {
+      /** only buildings can be demolished */
+      if (!fieldLabel?.includes('Building')) {
```

Comment:
This logic should also exist in the DB to ensure the user doesn't submit invalid JSON.  Additionally you should ensure only a SRES user `admin-properties` can selected these classifications in the backend.  Remember to never trust the frontend.

Reply:
Gotchya, makes sense